src/pages/Delegate/components/DelegateButton.tsx

```diff
@@ -20,10 +20,10 @@ export default function DelegateButton({ nft, delegatee }: { nft: GovNft; delega
       // @ts-ignore
       Toaster.toast(error);
     } else if (isConfirmed) {
-      navigate(`/nft/${nft.id}?collection=${nft.address}`);
+      navigate(`/nft/${nft.id}`);
```

Comment:
this (and other examples) will be the same as the query params should always be present